src/pretix/plugins/banktransfer/views.py

```diff
@@ -543,3 +584,223 @@ def order_qs(self):
                     organizer=self.request.organizer, can_change_orders=True, can_view_orders=True
                 ).values_list('limit_events__id', flat=True)
             )
+
+
+def _row_key_func(row):
+    return row['iban'], row['bic']
+
+
+def _unite_transaction_rows(transaction_rows):
+    united_transactions_rows = []
+    transaction_rows = sorted(transaction_rows, key=_row_key_func)
+    for (iban, bic), group in itertools.groupby(transaction_rows, _row_key_func):
+        rows = list(group)
+        united_transactions_rows.append({
+            "iban": iban,
+            "bic": bic,
+            "id": ", ".join(sorted(set(r['id'] for r in rows))),
+            "payer": ", ".join(sorted(set(r['payer'] for r in rows))),
+            "amount": sum(r['amount'] for r in rows),
+        })
+    return united_transactions_rows
+
+
+class RefundExportListView(ListView):
+    permission = 'can_change_orders'
+    template_name = 'pretixplugins/banktransfer/refund_export.html'
+    model = RefundExport
+    context_object_name = 'exports'
+
+    def get_success_url(self):
+        raise NotImplementedError
+
+    def get_unexported(self) -> QuerySet:
+        raise NotImplementedError()
+
+    def dispatch(self, request, *args, **kwargs):
+        if len(request.organizer.events.order_by('currency').values_list('currency', flat=True).distinct()) > 1:
+            messages.error(request, _('Please perform per-event refund exports as this organizer has events with '
+                                      'multiple currencies.'))
+            return redirect('control:organizer', organizer=request.organizer.slug)
+        return super().dispatch(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        ctx = super().get_context_data()
+        ctx['num_new'] = self.get_unexported().count()
+        ctx['basetpl'] = "pretixcontrol/event/base.html"
+        if not hasattr(self.request, 'event'):
+            ctx['basetpl'] = "pretixcontrol/organizers/base.html"
+        return ctx
+
+    @transaction.atomic()
+    def post(self, request, *args, **kwargs):
+        unite_transactions = request.POST.get("unite_transactions", False)
+        valid_refunds: Set[OrderRefund] = set()
+        for refund in self.get_unexported().select_related('order', 'order__event'):
+            if not refund.info_data:
+                # Should not happen
+                messages.warning(request,
+                                 _("We could not find bank account information for the refund {}. It was marked as failed.")
```

Comment:
In translatables, please always use a named parameter so the translator has a hint what will be inserted there.